### PR TITLE
Fix requirements for L 5.1

### PR DESCRIPTION
**Pull request by billmn**
This solve the the issue #10 ... now is possibile to use Dialect with Laravel 5.1

### CODE REVIEW

**Comment by billmn** [comment]
After Travis fix can I remove "configureComposer.sh" file? From what I see you used it only for Travis.

**Comment by darrylkuhn** [comment]
It is only used for travis, but it needs to be there for every travis run (which is every commit) so please don't remove it.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
+
+before_script:
+  - travis_retry composer self-update
+  - travis_retry composer update --no-interaction --prefer-source
```
Comment (darrylkuhn) [inline .travis.yml]: This should remain `composer install`. When the composer json is changed you can do a composer update on your local copy and then check in the composer.lock

Also what is the reason you changed `--prefer-dist` to `--prefer-source`?

**Comment by billmn** [comment]
Oh my wrong... I forgot prefer-source. 

I've changed to composer update because Travis give me an error on old dependencies. 

Today I fix it.

**Comment by billmn** [comment]
I suggest to create a L4 branch where store current code because Laravel 5.1 needs PHP >= 5.5.9 and Travis fail with this error on PHP 5.4:

<code>illuminate/contracts 5.1.x-dev requires php >=5.5.9 -> your PHP version (5.4.37) does not satisfy that requirement.</code>

schermata 2015-06-30 alle 10 34 49

Do you agree?

**Comment by darrylkuhn** [comment]
Hey I finally got around to taking care of this. @billmn per your suggestion I have dropped support for php 5.4

**Pull request closed without merging**